projects/gov-ui/src/lib/components/gov-uk-checkbox/gov-uk-checkbox.component.ts

```diff
@@ -21,7 +21,7 @@ import {FormGroup} from '@angular/forms';
 export class GovUkCheckboxComponent implements OnInit {
   constructor() { }
   @Input() group: FormGroup;
-  @Input() config: {value: string, label: string, hint: string; name: string; focusOn: string; id: string; classes: string};
+  @Input() config: {value: string, label: string, hint: string; name: string; focusOn: string; id: string; classes: string; spanText: string};
```

Comment:
This line is almost certainly too long for linting, which is usually set to 140 characters max per line. I think it'll fail.

Also, `spanText` seems to be something to do with a hidden accessibility descriptor. A name more along those lines would be a good idea, I think.

Reply:
Renamed to hiddenLabelContext also split the line to avoid linting issues